pkg/pipeline/sink/file.go

```diff
@@ -44,10 +43,8 @@ func (s *FileSink) Start() error {
 }
 
 func (s *FileSink) Close() error {
-	location, size, err := s.Upload(s.LocalFilepath, s.StorageFilepath, s.OutputType, false)
-	if err != nil {
-		return err
-	}
+	location, size, err := s.Upload(s.LocalFilepath, s.StorageFilepath, s.OutputType, false, "file")
+
```

Comment:
this error needs to be returned at some point

Reply:
fixed so the error is returned